### PR TITLE
deps(protractor): Upgrade Protractor to 4.0.11

### PR DESCRIPTION
**Pull request by heathkit**
**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Other... Please describe: Updating protractor to the latest
```

**What is the new behavior?**
Instead of being available in the global namespace, protractor's objects must now be imported.  For example,

```ts
import {browser, ExpectedConditions, $} from 'protractor';
```

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: modules/playground/e2e_test/key_events/key_events_spec.ts [hunk 1]
```diff
@@ -7,6 +7,11 @@
  */
 
 import {verifyNoBrowserErrors} from 'e2e_util/e2e_util';
+import {browser, by, element} from 'protractor';
+
+// TODO: Key exists on protractor; however, the typings for these are from
+// @types/selenium-webdriver which needs to be fixed.
```
Comment (juliemr) [inline modules/playground/e2e_test/key_events/key_events_spec.ts]: This TODO isn't very clear - can we rewrite to clarify what 'for these' means?

**Comment by juliemr** [comment]
Yay! LGTM once Travis is happy.

File: modules/playground/e2e_test/key_events/key_events_spec.ts (hunk 1 above)
Comment (mgiambalvo) [inline modules/playground/e2e_test/key_events/key_events_spec.ts]: The TODO is TODONE, so I just fixed it

**Comment by vicb** [comment]
Actually please fix the commit messages, see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

**Comment by vicb** [comment]
Oh I guess we can squash ! nevermind.

**Comment by vicb** [comment]
for some reason circleCi was not triggered. Opened a new PR fixing the formatting issues.

**Pull request closed without merging**